extension/src/fileSystem/index.test.ts

```diff
@@ -1,17 +1,28 @@
 import { join, relative, resolve } from 'path'
-import { ensureDirSync, remove } from 'fs-extra'
+import { appendFileSync, ensureDirSync, ensureFileSync, remove } from 'fs-extra'
 import {
   exists,
   findAbsoluteDvcRootPath,
   findDvcRootPaths,
   isDirectory,
   isSameOrChild,
-  getModifiedTime
+  getModifiedTime,
+  findOrCreateDvcYamlFile,
+  scriptCommand
 } from '.'
 import { dvcDemoPath } from '../test/util'
 import { DOT_DVC } from '../cli/dvc/constants'
 
 jest.mock('../cli/dvc/reader')
+jest.mock('fs-extra', () => {
+  const actualModule = jest.requireActual('fs-extra')
+  return {
+    __esModule: true,
+    ...actualModule,
+    appendFileSync: jest.fn(),
```

Comment:
[N] Could set this up with 
```
mockedAppendFileSync = jest.mocked(appendFileSync)
...
appendFileSync: mockedAppendFileSync
```

so that is is obvious that it is a mock if you only read the test.